dotnet/src/Extensions/Planning.ActionPlanner/FunctionViewExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.Planning.Action;
+
+internal static class FunctionViewExtensions
+{
+    /// <summary>
+    /// Create a manual-friendly string for a function.
+    /// </summary>
+    /// <param name="function">The function to create a manual-friendly string for.</param>
+    /// <returns>A manual-friendly string for a function.</returns>
+    internal static string ToManualString(this FunctionView function)
```

Comment:
Since we're duplicating this code, it might be worth preserving the previous formatting as much as possible (while also using `DefaultValue` to fix the bug) -- I know @dluc wanted to promote lower token usage for ActionPlanner. Curoius your thoughts, too.

Reply:
Added default value to existing `PopulateList` function